<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifContact</class>
 <widget class="QWidget" name="ModifContact">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modifier Contact</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Mail                 </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="MailEdit_2"/>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Telephone     </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="PhoneEdit_2"/>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Nom                </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="NomEdit_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Entreprise     </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="EntrepriseEdit_2"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Prenom          </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="PrenomEdit_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item alignment="Qt::AlignRight">
      <widget class="QPushButton" name="AnnulerButton_2">
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SupprimerButton">
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignLeft">
      <widget class="QPushButton" name="EnregistrerButton_2">
       <property name="text">
        <string>Enregistrer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
